kernel/libc/pthreads/pthread_mutex.c

```diff
@@ -6,6 +6,8 @@
 #include <errno.h>
 #include <assert.h>
 
+#include <kos/mutex.h>
```

Comment:
This ensures access to MUTEX_TYPE_NORMAL which isn't directly in the other includes.